docs/codeql/reusables/codespaces-template-note.rst

```diff
@@ -0,0 +1,6 @@
+.. pull-quote::
+
+   Note
+
+   For a guided introduction to CodeQL, we've created a CodeQL template (beta) in `GitHub Codespaces <https://github.com/codespaces/new?template_repository=github/codeql-codespaces-template>`__. You can use this template to test CodeQL concepts. However, if you would like to run CodeQL queries on code, you will need to install the CodeQL extension in Visual Studio Code. For instructions, see ":ref:`Setting up CodeQL in Visual Studio Code <setting-up-codeql-in-visual-studio-code>`."
```

Comment:
I'm not entirely happy with the phrasing of this note, so if anything comes to your mind that could be improved, I'd be grateful for suggestions! 🙇‍♀️

Reply:
Once we've worked out the wording, I think we might want to use this in place of the current reusable about VS Code [here](https://github.com/github/codeql/blob/siaramist/codeql-template/docs/codeql/writing-codeql-queries/find-the-thief.rst?plain=1#LL53): `.. include:: ../reusables/setup-to-run-tutorials.rst` to this new reusable.

Reply:
Thank you so much! 💖 I made a couple of tweaks to it in [094a9f4](https://github.com/github/codeql/pull/11554/commits/094a9f4aa742a10614c5ac0512d01501c1d9703a), just to make it fit in other articles too. I also overthought "language-agnostic" vs. "programming-language-agnostic" vs. wondering if that actually makes sense, so let me know if you have thoughts on that. 😅